content/docs/v3.4/dev-guide/_index.md

```diff
@@ -1,3 +1,5 @@
 ---
 title: Developer guide
+weight: 3000
```

Comment:
nit, let's say "Guide for developers to work with ectd" (or "etcd guide for developers") - trying to make it more generic (cover both who are using and or providing development contribution to the etcd project") @nate-double-u - thanks!

Reply:
Would you like me to change this in `next` as well?